### DISCUSSION TITLE
Upload URL to accompany the download URLs

### DISCUSSION THREAD

**Discussion by 4AloxeCorton** [NuSkooler/enigma-bbs#372] [Ideas]
Hi,

I've been playing around with this BBS package for the last few week on a custom docker ubuntu installation running on a synology NAS and I like what I see a lot.  It does a great job bridging the classic old school BBS world with some of the more modern advances such as the temporary URL download links available in parallel to the X/Y/Z modem.  However it would be great if there could also be an _Upload_ URL so that a user can completely forgo the X/Y/Z modem protocols which have their limitations and aren't that common anymore.
Just an idea for the backlog.

Cheers,

AA

**Comment by NuSkooler** [comment]
@4AloxeCorton Thanks for the feedback and I'm glad you're liking it! A HTTP upload point is something I'd definitely like to add as well when I get back to hacking on this.
